maia-hdl/projects/pluto/system_bd.tcl

```diff
@@ -194,18 +197,23 @@ ad_ip_parameter axi_ad9361 CONFIG.CMOS_OR_LVDS_N 1
 ad_ip_parameter axi_ad9361 CONFIG.MODE_1R1T 1
 ad_ip_parameter axi_ad9361 CONFIG.ADC_INIT_DELAY 21
 # parameters to reduce size
-ad_ip_parameter axi_ad9361 CONFIG.TDD_DISABLE 1
-ad_ip_parameter axi_ad9361 CONFIG.ADC_USERPORTS_DISABLE 1
-ad_ip_parameter axi_ad9361 CONFIG.ADC_DCFILTER_DISABLE 1
-ad_ip_parameter axi_ad9361 CONFIG.ADC_IQCORRECTION_DISABLE 1
-ad_ip_parameter axi_ad9361 CONFIG.DAC_DDS_DISABLE 1
-ad_ip_parameter axi_ad9361 CONFIG.DAC_USERPORTS_DISABLE 1
-ad_ip_parameter axi_ad9361 CONFIG.DAC_IQCORRECTION_DISABLE 1
+if {[info exists maia_iio]} {
 
+	ad_ip_parameter axi_ad9361 CONFIG.TDD_DISABLE 1
+	ad_ip_parameter axi_ad9361 CONFIG.DAC_DDS_DISABLE 1
+	ad_ip_instance maia_sdr_maia_iio maia_sdr
+} else {
+	ad_ip_parameter axi_ad9361 CONFIG.TDD_DISABLE 1
+	ad_ip_parameter axi_ad9361 CONFIG.ADC_USERPORTS_DISABLE 1
+	ad_ip_parameter axi_ad9361 CONFIG.ADC_DCFILTER_DISABLE 1
+	ad_ip_parameter axi_ad9361 CONFIG.ADC_IQCORRECTION_DISABLE 1
+	ad_ip_parameter axi_ad9361 CONFIG.DAC_DDS_DISABLE 1
+	ad_ip_parameter axi_ad9361 CONFIG.DAC_USERPORTS_DISABLE 1
+	ad_ip_parameter axi_ad9361 CONFIG.DAC_IQCORRECTION_DISABLE 1
+	ad_ip_instance maia_sdr_default maia_sdr
+}
 # Maia SDR core
 
-ad_ip_instance maia_sdr_default maia_sdr
-
```

Comment:
I suggest to move the common
```
ad_ip_parameter axi_ad9361 CONFIG.TDD_DISABLE 1
ad_ip_parameter axi_ad9361 CONFIG.DAC_DDS_DISABLE 1
```
outside of the if/else, and to add another if/else below the `# Maia SDR core` comment to select what IP core is instantiated as `maia_sdr`.